services/scan/src/precinct_scanner_app.ts

```diff
@@ -118,22 +109,15 @@ export async function buildPrecinctScannerApp(
   app.use(express.json({ limit: '5mb', type: 'application/json' }));
   app.use(express.urlencoded({ extended: false }));
 
-  app.get<NoParams, Scan.GetElectionConfigResponse>(
-    '/precinct-scanner/config/election',
-    (request, response) => {
-      const electionDefinition = store.getElectionDefinition();
-
-      if (request.accepts('application/octet-stream')) {
-        if (electionDefinition) {
-          response
-            .header('content-type', 'application/octet-stream')
-            .send(electionDefinition.electionData);
-        } else {
-          response.status(404).end();
-        }
-      } else {
-        response.json(electionDefinition ?? null);
-      }
+  app.get<NoParams, Scan.GetPrecinctScannerConfigResponse>(
+    '/precinct-scanner/config',
+    (_request, response) => {
+      response.json({
+        electionDefinition: store.getElectionDefinition(),
+        precinctSelection: store.getPrecinctSelection(),
+        markThresholdOverrides: store.getMarkThresholdOverrides(),
+        isSoundMuted: store.getIsSoundMuted(),
+      });
     }
   );
```

Comment:
I'm grouping all the config values together as one API request to simplify logic and reduce code on the backend and frontend.